test/e2e/readinessport_test.go

```diff
@@ -0,0 +1,58 @@
+package e2e
```

Comment:
we need to add these two lines at the top of the e2e test file
```
//go:build e2e
// +build e2e
```

Reply:
and the copyright as well

Reply:
Will do! Thanks